readme.md

```diff
@@ -221,6 +228,33 @@ You can optionally pass one or more of these parameters every time you use Lambo
   ```bash
   lambo new superApplication --full
 
+## GitHub Repository Creation
+
+- `--github=<options>` to create a new GitHub repository and push your new project to it.
+
+  ```bash
+  lambo new superApplication --github='--private'
+  ```
+
+Lambo uses the official GitHub command line tool to create your new repository so you must have it installed. You can find installation instructions [here](https://github.com/cli/cli#installation).
+
+New repository creation is configured by passing the value of the Lambo `--github` command line option directly to the GitHub command line tool. You must pass at least the desired visibility of your new repository, one of, `--private`, `--public`, or `--internal`.
```

Comment:
```suggestion
New repository creation is configured by passing the value of the Lambo `--github` command line option directly to the GitHub command line tool. You must pass at least the desired visibility of your new repository: `--private`, `--public`, or `--internal`.
```

Reply:
Since most folks, like me, wont' know what `internal` is, can we add a quick note?